src/SmartComponents/Group/add-group-modal.js

```diff
@@ -77,7 +77,8 @@ const AddGroupModal = ({
             isMulti={ true }
             placeholders={ 'Select Members' }
             options={ dropdownItems }
-            defaultValue={ initialValues ? initialValues.members.map(user => user.id) : [] }
+            defaultValue={ (initialValues && initialValues.members) ? initialValues.members.map(
```

Comment:
You can get rid of this condition if you set `initialValues` in component `defaultProps`. Unless there is a case when initial values can be object without `members` property,

Reply:
the groups can be created without the members property.